src/coreclr/jit/emitarm64.cpp

```diff
@@ -5493,11 +5520,24 @@ void emitter::emitIns_R(instruction ins, emitAttr attr, regNumber reg)
             break;
 
         case INS_mrs_tpid0:
+            assert(opt == INS_OPTS_NONE);
             id = emitNewInstrSmall(attr);
             id->idReg1(reg);
             fmt = IF_SR_1A;
             break;
 
+        case INS_sve_aesmc:
+        case INS_sve_aesimc:
+            assert(opt == INS_OPTS_SCALABLE_B);
+            id = emitNewInstrSmall(attr);
+            id->idInsOpt(opt);
+            id->idReg1(reg);
+            assert(insOptsScalable(id->idInsOpt()));
```

Comment:
i don't think we should have `opt` at all in all the previous cases. Instead just have something like:

```suggestion
            id->idInsOpt(INS_OPTS_SCALABLE_B);
            assert(insOptsScalable(INS_OPTS_SCALABLE_B));
```